tcf_website/management/commands/index_elasticsearch.py

```diff
@@ -0,0 +1,113 @@
+"""
+Modules used
+"""
+import json
+import requests
+
+from django.core.management.base import BaseCommand, CommandError
+from tcf_website.models import Course, Instructor
+
+class Command(BaseCommand):
+    """Indexes the Elastic AppSearch instance w/ Course and Instructor.
+
+    Author: Bradley Knaysi
+
+    Created: April 19th, 2020
+
+    How To Use:
+
+        $ cd theCourseForum2/
+        $ docker-compose up
+
+        open a new terminal
+
+        $ cd theCourseForum2/
+        $ docker exec -it tcf_django bash
+        $ python3 manage.py index_elasticsearch
+
+    WARNING: This should only be done by an Executive Team member once a semester
+    after new course and instructor data are added to the tcf_db. Also the Elastic AppSearch
+    portal takes like 10 minutes to fully update so be patient there. You can run this as
+    many times as you want! It updates a document in Elastic if it always exists so won't
+    double add or anything weird like that!
+
+    """
+
+    help = 'Erases and re-indexes the Elastic hosted Elasticsearch cluster'
+
+    def handle(self, *args, **options):
+
+        courses_engine_endpoint = 'https://1761244fa3674318b11f8d5729ffd7a2.app-search.us-east4.gcp.elastic-cloud.com/api/as/v1/engines/uva-courses/documents'
```

Comment:
You can move the endpoint and API key into the `.env` file and then access it like so:

`.env` file:
```bash
# .... current stuff above
ES_COURSE_ENDPOINT=https://1761244fa3674318b11f8d5729ffd7a2.app-search.us-east4.gcp.elastic-cloud.com/api/as/v1/engines/uva-courses/documents
ES_API_KEY=<key>
```

Then in `python` access them like this:
```python
import os

courses_engine_endpoint = os.environ['ES_COURSE_ENDPOINT']
api_key = os.environ['ES_API_KEY']
```

Reply:
gotcha

Reply:
Can you add those to this document: https://docs.google.com/document/d/1aAV0D8uD84h-JxjP_j7a2yKZz8pFM36P2tltHj_G3m8/edit

Reply:
- added production environment variables ✅ 
- updated google doc ✅ 
- tested by changing my environment to production ✅

Reply:
Can you split out those redundant lines into a function? After that I'll approve and merge